ci/blackbox/rdsbroker_test.go

```diff
@@ -118,6 +138,43 @@ var _ = Describe("RDS Broker Daemon", func() {
 				Expect(err).ToNot(HaveOccurred())
 				err = permissionsTest(credentials.URI)
 				Expect(err).ToNot(HaveOccurred())
+
+				By("restarting the broker with a new master password seed")
```

Comment:
If we do it outside, it means it would require to create 2x new instances and the tests would take longer in general. That's why I put it here, it would be faster.